src/cmds/restic/cmd_backup.go

```diff
@@ -264,7 +272,7 @@ func readBackupFromStdin(opts BackupOptions, gopts GlobalOptions, args []string)
 		return err
 	}
 
```

Comment:
I've actually thought about doing it this way, but the problem here is that during `init()` the flags are just registered, nothing is parsed yet. So `opts.Hostname` will always be empty.

I think we can leave it this way, having the hostname read at startup won't be noticeable. But good point (and good proposition)!